backend/main.go

```
@@ -2,8 +2,23 @@ package main
+	if err != nil {
+		log.Fatal("DB接続に失敗しました", err)
+	}
+
+	sqlDB, err := db.DB()
+	if err != nil {
+		log.Fatal("DBインスタンスの取得に失敗しました", err)
+	}
+
+	fmt.Println("DB接続に成功しました")
+
+	defer sqlDB.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `sqlDB.Close` is not checked

(errcheck)